src/masonite/api/providers/ResourceProvider.py

```diff
@@ -0,0 +1,20 @@
+"""A ResourceProvider Service Provider."""
+
+from masonite.provider import ServiceProvider
+from ..commands.ResourceCommand import ResourceCommand
+
+
+class ResourceProvider(ServiceProvider):
+    """Provides Services To The Service Container."""
+
+    wsgi = False
+
+    def register(self):
+        """Register objects into the Service Container."""
+        # self.app.bind('ResourceCommand', ResourceCommand())
```

Comment:
From other existing commands I choose `self.commands` to register the command but the documentation does not specify that but instead the commented line `self.app.bind`.
Could someone confirm, that `self.commands` is the way to go ?

Reply:
yes thats the way to go

Reply:
https://docs.masoniteproject.com/architectural-concepts/service-providers#commands